app/package.json

```diff
@@ -62,20 +52,19 @@
     "compression-webpack-plugin": "^2.0.0",
     "css-loader": "^3.2.0",
     "css-modules-typescript-loader": "^3.0.1",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
-    "enzyme-to-json": "^3.3.4",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.15.1",
+    "enzyme-to-json": "^3.4.3",
     "husky": "^4.2.3",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^23.4.1",
-    "jest-cli": "^23.4.1",
+    "jest": "^24.0.0",
```

Comment:
could update these versions to latest/later versions unless there's a specific dependency intended

Reply:
updated all jest related packages to latest